core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogBomScanner.java

```diff
@@ -74,6 +80,17 @@ public void unbind(ServiceReference<ManagementContext> managementContext) throws
 
     public class CatalogPopulator extends BundleTracker<Iterable<? extends CatalogItem<?, ?>>> {
 
+        private final String BROOKLYN_PACKAGE_REGEX = "org.apache.brooklyn.*";
+
+        private final ConfigKey<List<String>> WHITELIST =
+            ConfigKeys.newConfigKey(new TypeToken<List<String>>(){}, "brooklyn.catalog.osgi.application.whitelist",
+                "CSV Whitelist of patterns to match bundle symbolic id if bundle is to be permitted to add " +
+                    "applications (templates) to the catalog", ImmutableList.of(BROOKLYN_PACKAGE_REGEX));
+        private final ConfigKey<List<String>> BLACKLIST =
+            ConfigKeys.newConfigKey(new TypeToken<List<String>>(){}, "brooklyn.catalog.osgi.application.blacklist",
+                "CSV Blacklist of patterns to match bundle symbolic id to prevent selected whitelisted bundles " +
+                    "adding applications (templates) to the catalog", ImmutableList.<String>of());
+
```

Comment:
I guess it would be handy enough to add them in that file and then it could be referred to in the documentation (will do documentation as a separate PR).